### PR TITLE
upgrade to latest prometheus release 2.3.2 and alertmanager 0.15.1

### PR DESCRIPTION
**Pull request by davidkarlsen**
Upgrade to latest prometheus release

### CODE REVIEW

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: prometheus
-version: 6.8.1
-appVersion: 2.3.1
+version: 6.8.2
```
Comment (gianrubio) [inline stable/prometheus/Chart.yaml]: bump to major version, 6.9.0

Comment (davidkarlsen) [reply stable/prometheus/Chart.yaml]: Fixed (and another fix got in before, so now 6.10)

**Pull request merged**